Refine documentation related to logging (#265)
signalfx/signalfx-dotnet-tracing: docs/troubleshooting.md

> On Linux you can simply run: `cat /proc/<pid>/environ`

Reviewer: ```suggestion
On Linux, run: `cat /proc/<pid>/environ`
```

where `<pid>` is the Process ID.

> If all settings look fine and none of the suggestions above solves your issue,
> detailed logs are necessary. Follow the steps below to get detailed logs.

Reviewer: ```suggestion
Enable debug logging If none of the suggestions solved your issue. Follow these steps to enable debug logging for .NET instrumentation:
```

Set the environment variable `SIGNALFX_TRACE_DEBUG` to `true` before
the instrumented process starts.
By default, the library writes the log files under the below predefined locations.
If needed, change the default location by updating the environment variable
`SIGNALFX_TRACE_LOG_DIRECTORY` to an appropriate path.

On Linux, the default log location is `/var/log/signalfx/dotnet/`. If it does not

> exist, then run `/opt/signalfx/createLogPath.sh` to create it with appropriate permissions.

Reviewer: ```suggestion
exist,  run `/opt/signalfx/createLogPath.sh` to create it with the required permissions.
```
